models/models.js

```diff
@@ -1,15 +1,17 @@
 const db = require('../db/connection');
+const format = require('pg-format');
 
 exports.selectTopics = () => {
   return db.query('SELECT * FROM topics').then((topics) => {
     return topics.rows;
   });
 };
-// exports.selectTopics = async (req, res) => {
-//   try {
-//     const results = await db.query('SELECT * FROM topics')
-//     res.send(results.rows)
-//   } catch (err) {
-//     console.log(err);
-//   }
-// };
\ No newline at end of file
+
+exports.selectArticleById = (id) => {
+  return db.query('SELECT * FROM articles WHERE article_id = $1', [id])
```

Comment:
What we want to return here is a single article object. Right now you're returning an array of objects.